The GCC version is
```
$CC --version
powerpc64le-linux-gnu-gcc (GCC) 11.1.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

The errors I am seeing are specifically these:
```
[14:06:41] ../kernel/power/dgemm_small_kernel_nn_power10.c: In function ‘dgemm_small_kernel_nn_POWER10’:
[14:06:41] ../kernel/power/dgemm_small_kernel_nn_power10.c:180:3: warning: implicit declaration of function ‘__builtin_vsx_build_pair’; did you mean ‘__builtin_vsx_mul_2di’? [-Wimplicit-function-declaration]
[14:06:41]   180 |   __builtin_vsx_build_pair(&pair, (vec_t)v0, (vec_t)v1);
[14:06:41]       |   ^~~~~~~~~~~~~~~~~~~~~~~~
[14:06:41] ../kernel/power/dgemm_small_kernel_nn_power10.c:206:3: note: in expansion of macro ‘LOAD_PAIR’
[14:06:41]   206 |   LOAD_PAIR(pb0, t0, t1);           \
[14:06:41]       |   ^~~~~~~~~
[14:06:41] ../kernel/power/dgemm_small_kernel_nn_power10.c:368:13: note: in expansion of macro ‘LOAD_BTP_8x2’
[14:06:41]   368 |             LOAD_BTP_8x2(n, k);
[14:06:41]       |             ^~~~~~~~~~~~
[14:06:41] ../kernel/power/dgemm_small_kernel_nn_power10.c:368:13: error: AltiVec argument passed to unprototyped function
[14:06:41] ../kernel/power/dgemm_small_kernel_nn_power10.c:368:13: error: AltiVec argument passed to unprototyped function
[14:06:41] ../kernel/power/dgemm_small_kernel_nn_power10.c:368:13: error: AltiVec argument passed to unprototyped function
[14:06:41] ../kernel/power/dgemm_small_kernel_nn_power10.c:368:13: error: AltiVec argument passed to unprototyped function
[14:06:41] ../kernel/power/dgemm_small_kernel_nn_power10.c:379:13: error: AltiVec argument passed to unprototyped function
[14:06:41]   379 |             LOAD_BTP_8x1(n, k);
[14:06:41]       |             ^~~~~~~~~~~~
[14:06:41] ../kernel/power/dgemm_small_kernel_nn_power10.c:379:13: error: AltiVec argument passed to unprototyped function
[14:06:41] ../kernel/power/dgemm_small_kernel_nn_power10.c:405:11: error: AltiVec argument passed to unprototyped function
[14:06:41]   405 |           LOAD_BTP_8x2(n, k);
[14:06:41]       |           ^~~~~~~~~~~~
[14:06:41] ../kernel/power/dgemm_small_kernel_nn_power10.c:405:11: error: AltiVec argument passed to unprototyped function
[14:06:41] ../kernel/power/dgemm_small_kernel_nn_power10.c:405:11: error: AltiVec argument passed to unprototyped function
```

--- a/kernel/power/dgemm_small_kernel_nn_power10.c
+++ b/kernel/power/dgemm_small_kernel_nn_power10.c
@@ -167,7 +167,7 @@ typedef __vector unsigned char vec_t;
 
 #define INIT_1ACC() __builtin_mma_xxsetaccz(&acc0);
 
-#if (defined(__GNUC__) && (__GNUC__ == 10))
+#if (defined(__GNUC__) && (__GNUC__ == 10 || (__GNUC__ == 11 && __GNUC_MINOR__ <= 2)))
 #if defined(_AIX)
 #define LOAD_PAIR(pair, v0, v1)                             \
   __builtin_vsx_assemble_pair(&pair, (vec_t)v0, (vec_t)v1);

--- a/kernel/power/dgemm_small_kernel_tn_power10.c
+++ b/kernel/power/dgemm_small_kernel_tn_power10.c
@@ -167,7 +167,7 @@ typedef __vector unsigned char vec_t;
 
 #define INIT_1ACC() __builtin_mma_xxsetaccz(&acc0);
 
-#if (defined(__GNUC__) && (__GNUC__ == 10))
+#if (defined(__GNUC__) && (__GNUC__ == 10 || (__GNUC__ == 11 && __GNUC_MINOR__ <= 2)))
 #if defined(_AIX)
 #define LOAD_PAIR(pair, v0, v1)                             \
   __builtin_vsx_assemble_pair(&pair, (vec_t)v0, (vec_t)v1);

